src/main/play-doc/operation/DynamicProxyConfiguration.md

```diff
@@ -0,0 +1,627 @@
+
+## Requirement
+
+ConductR makes the assumption that an endpoint must be uniquely identified by its name given a particular system and system version.
+
+The endpoints that failed to satisfy this requirement will be considered as duplicate, and the duplication is assumed to be a result of configuration error. Because of this, duplicated endpoints will not be exposed via proxy configuration.
+
+## Creating and deploying custom HAProxy configuration
+
+It is possible to provide custom HAProxy configuration by supplying configuration overrides to the ConductR HAProxy bundle.
+
+[ConductR CLI](CLI) is required to perform the following steps, and as such it needs to be installed beforehand.
+
+These are the steps to deploying custom HAProxy configuration.
+
+### Setup
+
+Create a directory where the custom HAProxy configuration will be placed, e.g.
+
+```
+mkdir -p /tmp/custom-haproxy-conf
+```
+
+### Populate the custom HAProxy configuration template
+
+Create the custom HAProxy template, e.g.
+
+```bash
+touch /tmp/custom-haproxy-conf/haproxy-override.cfg
+```
+
+The file can be given any name as long as it's within the directory created from the previous step.
+
+In this example, we'll populate the file with the following custom template.
+
+```
+# HAProxy Specific Configuration
+global
+	log /dev/log	local0
+	log /dev/log	local1 notice
+	chroot /var/lib/haproxy
+	stats socket /run/haproxy/admin.sock mode 660 level admin
+	stats timeout 30s
+	user haproxy
+	group haproxy
+	daemon
+
+defaults
+	log	global
+	mode	http
+	option	httplog
+	option	dontlognull
+    timeout connect 5000
+    timeout client  50000
+    timeout server  50000
+
+
+# Custom Endpoint Configuration
+{{#eachAcls bundles defaultHttpPort=9443}}
+  {{#ifAcl 'public-site' '1' 'main'}}
+frontend my_www_frontend
+  bind {{haproxyHost}}:80
+  # Uncomment the following line if SSL is used
+  # bind {{haproxyHost}}:443 ssl crt {{overrideConfigDir}}/test-cert.pem
+  mode http
+  acl my_www_host_match hdr(host) -i www.acme.com
+  use_backend my_www_backend if my_www_host_match
+
+backend my_www_backend
+  mode http
+    {{#eachBackendServer}}
+  server {{serverName}} {{host}}:{{port}} maxconn 1024
+    {{/eachBackendServer}}
+  {{/ifAcl}}
+{{/eachAcls}}
+```
+
+Here are the explanation for the example template above:
+
+* *HAProxy Specific Configuration* section is declared prior *Custom Endpoint Configuration* section.
+* *HAProxy Specific Configuration* section is populated with operating system specific setup. In the example above it is populated with a cut-down version of Ubuntu's configuration.
+* *Custom Endpoint Configuration* section contains application specific proxy configuration.
+* The endpoint that matches the following criteria has a custom configuration declared within the [ifAcl](#ifAcl) block:
+  * `system` is `public-site`
```

Comment:
The child bullet points are currently not grouped under the parent bullet point.